package.json

```diff
@@ -150,6 +150,11 @@
           "scope": "window"
```

Comment:
Please change the description to "Enable/disable autoclosing of XML tags. \n\nIMPORTANT: Turn off editor.autoClosingTags for this to work", to make it consistent with other "autoclosing" preferences in VS Code.